src/status_im/extensions/capacities/views.cljs

```diff
@@ -0,0 +1,13 @@
+(ns status-im.extensions.capacities.views
+  (:require-macros [status-im.utils.views :refer [defview letsubs]])
+  (:require [status-im.ui.components.react :as react]
+            [status-im.ui.components.status-bar.view :as status-bar]
+            [status-im.ui.screens.profile.photo-capture.styles :as styles]))
+
+(defview screen-holder []
+  (letsubs [{:keys [view]} [:get-screen-params]]
+    (when view
+      (let [[_ _ reactview] (view)]
+        [react/view styles/container
+         [status-bar/status-bar]
+         reactview]))))
```

Comment:
no toolbar?

Reply:
we agreed with @jeluard to leave extension developer a full screen and the freedom to place the close button where he prefers...

Reply:
Right this is fullscreen accessible to users, so nothing. It's up to end dev to decide what's there.